assembly/index.ts

```diff
@@ -1,3 +1,33 @@
+const welcomePage = `<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Welcome to AS on Compute@Edge</title>
```

Comment:
The title across the other starter kits are also consistent...

- https://github.com/fastly/compute-starter-kit-javascript-default/blob/main/src/welcome-to-compute%40edge.html#L10
- https://github.com/fastly/compute-starter-kit-rust-default/blob/main/src/welcome-to-compute%40edge.html#L10

Reply:
UPDATE: I've just seen there are identical PRs added to the other language starter kits. I personally see no value in adding the runtime to the title of this synthetic response. @triblondon thoughts?

Reply:
Oh, I just reviewed the Rust one. I assumed the PR was a mistake until I saw these.

Agreed that this change is redundant.